components/folder/NavBox.tsx

```diff
@@ -53,26 +58,24 @@ const NavBox = ({ navId, onClickNavItem }: FolderNav) => {
   return (
     <>
       <S.NavWrapBox>
-        <ul>
-          <NavItem
-            navName="전체"
-            onClick={onClickNavItem}
-            navId={navEntireTab}
-            isCurrentNav={navId === navEntireTab}
-          />
+        <S.NavList>
+          <NavItem navName="전체" isCurrentNav={!pageNavId} />
           {navList &&
             navList.map((navItem) => {
               return (
                 <NavItem
                   key={navItem.id}
                   navName={navItem.name}
                   navId={navItem.id}
-                  onClick={onClickNavItem}
-                  isCurrentNav={navItem.id === navId ? true : false}
+                  isCurrentNav={
+                    pageNavId && navItem.id?.toString() === pageNavId[0]
+                      ? true
+                      : false
+                  }
```

Comment:
조건문 자체가 boolean 이므로 간단하게 변경해볼수있겠네요
```suggestion
                  isCurrentNav={
                    pageNavId && navItem.id?.toString() === pageNavId[0]
                  }
```